README.md

```
@@ -33,11 +53,23 @@ Once these variables are saved in your Repository secrets, your commits will be
+export COMMITER_EMAIL=your_email@example.com
+...
+```
 3. Run the tool locally whenever you want to sync your activity.
 
+## Configuration
+This project uses GitHub Actions to automate builds and daily synchronization:
+
+•	GitHub Actions Workflow: The .github/workflows/schedule.yml defines the automation steps for building and running the tool.
+•	Secrets Configuration: The secrets allow secure storage and retrieval of required tokens and URLs during automation.
+
 ### Important Notes:
```

Markdownlint

Punctuation: ':'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)